src/Organizer/Organizer.php

```diff
@@ -344,4 +377,28 @@ private function getTitle(\CultureFeed_Cdb_Item_Actor $actor)
             return null;
         }
     }
+
+    /**
+     * @param Title $title
+     * @param Language $language
+     */
+    private function setTitle(Title $title, Language $language)
+    {
+        $this->titles[$language->getCode()] = $title;
+    }
+
+    /**
+     * @param Title $title
+     * @param Language $language
+     * @return bool
+     */
+    private function isTitleChanged(Title $title, Language $language)
+    {
+        if (!isset($this->titles[$language->getCode()]) ||
```

Comment:
Dit kan je eventueel aanpassen naar `return !isset($this->titles[$language->getCode()]) || !$title->sameValueAs($this->titles[$language->getCode()])` en de `if/else` en `return true/false` dan weg laten.